src/common/transformations/src/transformations/transpose_sinking/ts_squeeze.cpp

```diff
@@ -102,38 +103,52 @@ TSSqueezeForward::TSSqueezeForward() {
     MATCHER_SCOPE(TSSqueezeForward);
 
     auto transpose_label = wrap_type<Transpose>({any_input(), wrap_type<Constant>()});
+    auto squeeze_with_1_input = wrap_type<Squeeze>({transpose_label});
```

Comment:
Agree, this is a bug, we can solve it without code change if enable TS in MOC pipeline (ticket: 108227). But this requires significant effort.
if we disable the case it will break some model performance (need to double-check ticket for this Release: 108231)

Reply:
Regarding `Squeeze` with one input, I think we need to solve this problem in this release.